Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -1069,6 +1087,31 @@ void MainWindow::OnExportRecording()
   Movie::SaveRecording(dtm_file.toStdString());
 }
 
+void MainWindow::ShowTASInput()
+{
+  for (int i = 0; i < 4; i++)
```

Comment:
These `4`s can be replaced with the numeric constants that were added to the class. Ditto in other relevant locations